docker/Dockerfile-xenial

```diff
@@ -1,31 +1,24 @@
 # -*- mode: dockerfile -*-
 
-FROM nvidia/cuda:10.2-cudnn7-devel-ubuntu16.04
```

Comment:
Is there a specific reason for moving to standard ubuntu images? I built everything on Nvidia's images so that people would get GPU support out of the box.